util/dvsim/FlowCfg.py

```diff
@@ -393,20 +396,41 @@ def gen_results(self, results):
 
         '''
         for item in self.cfgs:
+            item.report_path = os.path.join(item.scratch_path, "report")
+            item.create_report_dir(item.report_path)
             result = item._gen_results(results)
             log.info("[results]: [%s]:\n%s\n", item.name, result)
             log.info("[scratch_path]: [%s] [%s]", item.name, item.scratch_path)
+            log.info("[report_path]: [%s] [%s]", item.name, item.report_path)
             self.errors_seen |= item.errors_seen
 
+        self.create_report_dir(self.report_summary_path)
         if self.is_primary_cfg:
             self.gen_results_summary()
+            self.write_results_summary_html()
         self.gen_email_html_summary()
```

Comment:
@sriyerg I think for next PR, we can remove the email_html_summary.
We can output all summaries in html and let user decide which one to use for emailing.

To accomplish this, I think we can create the latest output files without the time/date postfix, and use the script to rename previous output. So user will know which file has the latest output.